resolwe/flow/managers/dispatcher.py

```diff
@@ -30,6 +30,7 @@
 
 from resolwe.flow.engine import InvalidEngineError, load_engines
 from resolwe.flow.execution_engines import ExecutionError
+from resolwe.flow.executors.constants import DATA_ALL_VOLUME, SECRETS_VOLUME
```

Comment:
"Prepore secrets and config for data transfer
inside communication container."

This is quite a generic commit message. Try to improve it, and preferably put it into a single line.